build_tools/benchmarks/run_benchmarks_on_android.py

```diff
@@ -559,7 +559,7 @@ def set_cpu_frequency_scaling_governor(governor: str):
 def set_gpu_frequency_scaling_policy(policy: str):
   git_root = execute_cmd_and_get_output(["git", "rev-parse", "--show-toplevel"])
   device_model = get_android_device_model()
-  if device_model == "Pixel-6":
+  if device_model == "Pixel-6" or device_model == "Pixel-6-Pro":
```

Comment:
Huh, I thought we were just going to get the regular ones. Ah well.

Reply:
Yeah, I was under the same impression. But the device said otherwise. :)